### PR TITLE
Pc/pandas

### PR DESCRIPTION
**Pull request by Blaizzy**
# Description

Include a summary of the changes and the related issue.

__Related to:__ <ClickUp/JIRA task name>

__Any expected test failures?__

---

Add a `[X]` to relevant checklist items

## ❔ This change

- [x] adds a new feature
- [ ] fixes breaking code
- [ ] is cosmetic (refactoring/reformatting)

---

## ✔️ Pre-merge checklist

- [x] Refactored code ([sourcery](https://sourcery.ai/))
- [x] Tested code locally
- [x] Precommit installed and run before pushing changes
- [x] Added code to GitHub tests ([notebooks](workflows/test-notebooks.yml), [scripts](workflows/test-scripts.yml))
- [x] Updated GitHub [README](../README.md)
- [ ] Updated the projects overview page on Notion

---

## 🧪 Test Configuration

- OS:
- Python version:
- Neptune version:
- Affected libraries with version:

### CODE REVIEW

**Review by SiddhantSadangi — DISMISSED** [review]

**Comment by SiddhantSadangi** [comment]
@TytusKalicki - can you check? we'll need some changes in the docs too

File: integrations-and-supported-tools/pandas/notebooks/Neptune_Pandas.ipynb [hunk 1]
```diff
@@ -0,0 +1,429 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "C73PO_vy-cbf"
+   },
+   "source": [
+    "# Using Neptune with pandas\n",
+    "[pandas](https://pandas.pydata.org/)  is a popular open-source data analysis and manipulation tool. With Neptune, you can log and visualize pandas DataFrames.\n",
```
Comment (TytusKalicki) [inline integrations-and-supported-tools/pandas/notebooks/Neptune_Pandas.ipynb:10]: there's a double space here

**Review by TytusKalicki — APPROVED** [review]
all good! I noticed the new example runs in app - will add them to the docs :)

**Comment by SiddhantSadangi** [comment]
The new showcase run, and also the optional section about uploading dataframe profile reports to Neptune

**Pull request merged**